framework_tvb/tvb/adapters/analyzers/fourier_adapter.py

```diff
@@ -214,15 +243,33 @@ def launch(self, view_model):
         fft_index.ndim = len(spectra_file.array_data.shape)
         input_time_series_h5.close()
 
-        fft_index.windowing_function = self.algorithm.window_function
-        fft_index.segment_length = self.algorithm.segment_length
-        fft_index.detrend = self.algorithm.detrend
+        fft_index.windowing_function = view_model.window_function
```

Comment:
We should be able to replace now this block that populates `fft_index` with `fft_index.fill_from_has_traits(partial_result)`